Made event post for techtalk with Monica Beckwith

fat1996 opened the pull request.

fat1996 committed:
Made event post for techtalk with Monica Beckwith

evykassirer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
++Title: Techtalk with Monica Beckwith
```
it's not showing up on the test site because of these `+`s at the start of your lines

fat1996 replied:
Removed the extra +'s.

fat1996 committed:
fixed the format of the file

fat1996 commented:
Removed the extra +'s.

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
```
Maybe something like "Monica Beckwith will be visiting campus to give a talk on her work in ...". You can have a look at https://github.com/wics-uw/website/blob/master/content/W2015/W15-event-hypatia.md for inspiration.

evykassirer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+Title: Techtalk with Monica Beckwith
```
I think it should be two words "tech talk"

annalorimer replied:
Techtalk -> Tech Talk

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
+
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade not just optimizing the JVM heuristics, but also improving the Just-in-time(JIT) code quality for various processor architectures as well as working with the garbage collectors and improving garbage collection for server systems.
```
80 chars per line.

evykassirer replied:
lines should be max 80 characters

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade not just optimizing the JVM heuristics, but also improving the Just-in-time(JIT) code quality for various processor architectures as well as working with the garbage collectors and improving garbage collection for server systems.
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she will dive into the HotSpot garbage collection and provide an overview of the various garbage collectors available in HotSpot.
```
80 chars per line.

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+
+## Event Details ##
+
++ **Who:** This is a women-only event. 
```
Need to link to FAQ. Edit: Oh actually this is open to everyone!

evykassirer replied:
it's open to everyone!

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
++ **Who:** This is a women-only event. 
++ **What:** Come join us for a talk with Monica Beckwith
++ **Where**: STC 0040
++ **When:** Tuesday March 14, 6:00-7.30pm
```
7.30pm -> 7:30pm

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+## Event Details ##
+
++ **Who:** This is a women-only event. 
++ **What:** Come join us for a talk with Monica Beckwith
```
This should be the name of the talk

annalorimer commented on content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:
```diff
@@ -0,0 +1,18 @@
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
```
This is a little long now 😛

fat1996 committed:
Made all the requested changes.

Fixed the formatting of the post

Reduced line size to 80 chars

Updated the summary
